### PR TITLE
added and updated COD rivers and seaports functionality 

### PR DESCRIPTION
**Pull request by SteveP2000**
Includes fix to validation process.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -10,25 +10,29 @@
     install_requires=[],
     entry_points={
         'console_scripts': [
-            # Boundaries
             'extract_adm0_cod=extract.cod:get_adm0_snakemake',
             'extract_adm1_cod=extract.cod:get_adm1_snakemake',
             'extract_adm2_cod=extract.cod:get_adm2_snakemake',
             'extract_adm3_cod=extract.cod:get_adm3_snakemake',
+            'extract_roads_cod=extract.cod:get_roads_snakemake',
```
Comment (turnerm) [inline setup.py:17]: The roads COD entrypoints are already defined on lines 36 and 36, the duplicates need to be removed or else `pip install .` fails

File: snakefile [hunk 2]
```diff
@@ -122,6 +138,33 @@ rule transform_adm3_cod:
     shell:
         "transform_adm3_cod {input} {output}"
 
+rule transform_roads_cod:
```
Comment (turnerm) [inline snakefile:141]: Already defined later in this file

**Comment by SteveP2000** [comment]
@turnerm I believe everything is included in this pull request? All schema checks appear to be present in this one.
Anything else required to prevent conflicts?

**Comment by SteveP2000** [comment]
@turnerm have removed duplicate roads from snakemake file as requested

**Pull request merged**